Kickstarter-iOS/Views/Controllers/RewardsCollectionViewController.swift

```diff
@@ -87,23 +89,22 @@ final class RewardsCollectionViewController: UICollectionViewController {
     self.viewModel.inputs.viewDidAppear()
   }
 
-  override func viewDidLayoutSubviews() {
-    super.viewDidLayoutSubviews()
-
-    guard let layout = self.flowLayout else { return }
-
-    let itemSize = self.calculateItemSize(from: layout, using: self.collectionView)
-
   override func viewWillAppear(_ animated: Bool) {
     super.viewWillAppear(animated)
 
     self.viewModel.inputs.viewWillAppear()
   }
 
-  override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
-    super.viewWillTransition(to: size, with: coordinator)
+  override func viewDidLayoutSubviews() {
```

Comment:
Hmm this shouldn't be new, it's in master. I think you need to pull in the latest from master.

Reply:
Yeah I merged, I think it just moved in the merge. lemme move it above `viewWillAppear`, should make it go away.